.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: c
+sudo: false
+branches:
+  except:
+    - /^(wip\/)?(appveyor|msvc|mingw|windows)(\-.+)?$/
+
+matrix:
+  include:
+    - env: C_COMPILER=gcc-6 CXX_COMPILER=g++-6
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - gcc-6
+          - g++-6
+          - nasm
```

Comment:
OK, that's cool you can just say that you want the `nasm` package and it will be there when the CI runs? I wonder how it works with versions: I've updated to the newest nasm version since it fixes some bugs that were affecting me - but it was only in the debuginfo generation so it probably doesn't matter for CI.

Reply:
It just installs the [nasm package](https://packages.ubuntu.com/trusty/nasm). Travis is running on Ubuntu 14.04 LTS.

Reply:
OK, yeah that one is buggy but I guess it won't affect CI.